xanadu_sphinx_theme/__init__.py

```diff
@@ -25,7 +26,7 @@ def setup(app):
         "details": DetailsDirective,
         "gallery-item": GalleryItemDirective,
         "index-card": IndexCardDirective,
-        "related-demo": RelatedDemoDirective,
+        "related": RelatedDemoDirective,
```

Comment:
Doesn't `related-demo` match the naming of the other directives better? 🤔

Reply:
This was me sneaking in a feature 😬 I realized that the sidebar section just says "Related", and I realized this could be useful beyond just demos

Reply:
Should we rename the directive then too, @josh146?
```suggestion
        "related": RelatedDirective,
```

Reply:
Oops, I should have done that at the time